validator/accounts/v2/export_test.go

```diff
@@ -0,0 +1,117 @@
+package v2
+
+import (
+	"context"
+	"flag"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/shared/testutil/assert"
+	"github.com/prysmaticlabs/prysm/validator/flags"
+	"github.com/urfave/cli/v2"
+
+	"github.com/prysmaticlabs/prysm/shared/testutil"
+	v2 "github.com/prysmaticlabs/prysm/validator/keymanager/v2"
+)
+
+func setupWallet(t *testing.T, testDir string) *Wallet {
+	walletDir := filepath.Join(testDir, walletDirName)
+	passwordsDir := filepath.Join(testDir, passwordDirName)
+	ctx := context.Background()
+	err := initializeDirectWallet(walletDir, passwordsDir)
+	assert.NoError(t, err)
+	cfg := &WalletConfig{
+		WalletDir:      walletDir,
+		PasswordsDir:   passwordsDir,
+		KeymanagerKind: v2.Direct,
+	}
+	w, err := NewWallet(ctx, cfg)
+	if err != nil {
```

Comment:
No need for this, you can do `require.NoError(t, err)` which does the same as if err != nil { t.Fatal(err